app/adapters/parameter.js

```diff
@@ -0,0 +1,58 @@
+import DS from 'ember-data';
+import ENV from '../config/environment';
+
+const { JSONAPIAdapter } = DS;
+
+export default JSONAPIAdapter.extend({
+
+    caxe: Ember.inject.service(),
+    ajax(url, method, hash) {
+        hash = hash || {};
+        hash.headers = hash.headers || {};
+        return this._super(url, method, hash);
+    },
+
+    // Polyfill queryRecord
+    queryRecord(store, type, query) {
+
+        var url = this.buildURL(type.modelName, null, null, 'queryRecord', query) + '/';
+
+        console.log(url)
+
+        if (this.sortQueryParams) {
+            query = this.sortQueryParams(query);
+        }
+
+        return this.ajax(url, 'GET', { data: query })
+            .then(function(result) {
+                result = result.data;
+                // hack to fix https://github.com/emberjs/data/issues/3790
+                // and https://github.com/emberjs/data/pull/3866
+                try {
+                    store.push({data: null});
+                    return {data: result || null};
+                } catch(e) {
+                    return {data: result || []};
+                }
+            }, function(result) {
```

Comment:
On error return empty

Reply:
this could deffo be a comment in the code